weather_mv/loader_pipeline/ee.py

```diff
@@ -312,7 +337,10 @@ def expand(self, paths):
                         asset_location=self.asset_location,
                         ee_asset_type=self.ee_asset_type,
                         open_dataset_kwargs=self.xarray_open_dataset_kwargs,
-                        disable_grib_schema_normalization=self.disable_grib_schema_normalization))
+                        disable_grib_schema_normalization=self.disable_grib_schema_normalization,
+                        band_names=band_names_dict,
```

Comment:
Consider renaming `band_names` to `band_names_dict` or some other name -- name of the variable should be self-sufficient in understanding its content.

Reply:
Resolved